app/services/participants/withdraw/validate_and_change_state.rb

```diff
@@ -16,6 +16,11 @@ def perform_action!
         ActiveRecord::Base.transaction do
           ParticipantProfileState.create!(participant_profile: user_profile, state: ParticipantProfileState.states[:withdrawn], reason: reason)
           user_profile.training_status_withdrawn!
+
+          unless user_profile.npq?
+            induction_coordinator = user_profile.school&.induction_coordinator_profiles&.first
+            SchoolMailer.fip_provider_has_withdrawn_a_participant(withdrawn_participant: user_profile, induction_coordinator: induction_coordinator).deliver_later
```

Comment:
Since we moved to sidekiq we should always schedule email outside of DB transactions as the worker can pick up scheduled job before the transaction is committed and sending email without all the data,

Reply:
Good spot! i noticed Tony did that for Partnership Challenge email on Friday, hadn't spotted it in my own.